yarn.lock

```diff
@@ -2074,13 +2074,14 @@
     npmlog "^4.1.2"
     write-file-atomic "^2.3.0"
 
-"@material-ui/data-grid@^4.0.0-alpha.6":
-  version "4.0.0-alpha.6"
-  resolved "https://registry.yarnpkg.com/@material-ui/data-grid/-/data-grid-4.0.0-alpha.6.tgz#78e14791777e1aa12b9b4b84d82d8e5ff437beb3"
-  integrity sha512-XFUdIK+It5PY3ivO4u0hDUxQNC4ASTuQHZlVe9K3RFEjZRNt70W29WoL36IvJ1o/bN208uUuQvzY9tn3J0P8lQ==
+"@material-ui/data-grid@4.0.0-alpha.26":
+  version "4.0.0-alpha.26"
```

Comment:
The latest version doesn't work because of https://github.com/mui-org/material-ui-x/pull/1499.